pkg/sql/opt/optbuilder/testdata/inverted-indexes

```diff
@@ -0,0 +1,181 @@
+# This file contains tests on tables with inverted indexes. Even though index
```

Comment:
Nice-to-have (but annoying to do) would be to have a new commit that is just the new tests. Then the current commit after it could demonstrate that things either do or do not change as expected.